Makefile

```diff
@@ -106,7 +106,7 @@ endif
 
 check-qt-dir:
 ifeq ($(shell qmake -v 2>/dev/null),)
```

Comment:
Hmm.. how does this differentiate between Qt5 and Qt6 qmake? For example, on Fedora:
```
❯ qmake
qmake         qmake-qt5     qmake-qt5.sh  qmake-qt6.sh  qmake6
```

So for me, there would be no way to explicitely state that I want to build the Qt6 version

Reply:
IMO we should have some switch here to choose; and prefer the Qt6 build by default

Reply:
UPD: `qmake` alone actually resolves to `qmake6`... but what if it doesn't?

Reply:
Ah I know! The system env that QtCreator uses for the build explicitely says:
```
QTDIR=/usr/lib64/qt5
```
and I guess that's where it picks the Qt version from

Reply:
hmm, in `Makefile` it's just like e.g.:
```
export QT5_LIBDIR := $(shell qmake -query QT_INSTALL_LIBS 2>/dev/null)
```
so in the end you should have proper `qmake` in the `PATH`, maybe QtCreator also handles that

Reply:
Yeah, maybe it does some extra magic there